README.md

```diff
@@ -97,7 +104,7 @@ the common operations for data processing and model training.
 
 * Optimizers from allennlp: Implemented as children of the base `Optimizer` class.
 
-* Metric computation is supported through `jury` library. In order to fully utilize transformers' Trainer's metric computation we introduced `MetricHandler`. The default behavior of this class is to directly decode the prediction outputs and use these for metric computation which is viable for most NLP tasks; however, for some tasks in which the prediction outputs are not suitable for the given metric by default behavior, you may need to extend this class for your own task. 
+* Metric computation is supported through `jury`. In order to make the metrics flexible enough to work with the trainer in a common interface, we introduced `MetricHandler`. The default behavior of this class is to directly decode the prediction outputs and use these for metric computation which is viable for most NLP tasks; however, for some tasks in which the prediction outputs are not suitable for the given metric by default behavior, you may need to extend this class for your own task. 
```

Comment:
3 problems.
1) We added the `postprocess` method to this class as well. However, this paragraph is only talking about the `preprocess` method like it is the only (or the main) thing this class performs.
2) `The default behavior of this class is to directly decode` -> This is incorrect since we changed the default behavior. Moreover, problem-1 applies here as well, this is the default behavior of the `preprocess` method, not the class.
3) (Minor one): Instead of `prediction outputs`, let's use somethind like `model predictions`.